trestle/core/profile_resolver.py

```diff
@@ -223,9 +257,55 @@ def _merge_controls(self, dest: cat.Control, src: cat.Control) -> None:
                 if not dest.controls:
                     dest.controls = src.controls
                 else:
-                    self._merge_lists(dest.controls, src.controls, prof.Method.merge)
-
-        def _merge_lists(
+                    self._merge_control_lists(dest.controls, src.controls, prof.Method.merge)
+
+        def _merge_params(self, dest: common.Parameter, src: common.Parameter) -> None:
+            """Merge src parameter into dest when the merge method is merge."""
+            if src.props is not None:
+                dest.props = as_list(dest.props)
+                dest_names = [prop.name for prop in dest.props]
+                for prop in src.props:
+                    if prop.name not in dest_names:
+                        dest.props.append(prop)
+            if src.links is not None:
+                dest.links = as_list(dest.links)
+                dest_hrefs = [link.href for link in dest.links]
+                for link in src.links:
+                    if link.href not in dest_hrefs:
+                        dest.props.append(link)
```

Comment:
The code pattern is the same but I'm doing this because the underlying structures accessed are different.  Let me know if you had an approach in mind.  prop.name vs. link.href